src/main/java/user/delete_comment/DeleteCommentInteractor.java

```diff
@@ -0,0 +1,80 @@
+package user.delete_comment;
+
+import user.delete_comment.exceptions.NotOwnCommentException;
+import entity.Comment;
+import exceptions.CommentNotFoundException;
+import exceptions.NotModeratorException;
+
+import java.util.ArrayList;
+
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+
+public class DeleteCommentInteractor implements IDeleteCommentInput{
```

Comment:
Don't forget to write tests! Otherwise, interactor looks good.

Reply:
Thank you for the reminder!